app/models.py

```diff
@@ -50,6 +50,7 @@ def serialize(self):
 
         return {
             'id': self.id,
+            'objectID': self.id,
```

Comment:
Algolia uses this property name on indexing.

Reply:
The quick fix for the tests is to add `'objectID': None,` right here
https://github.com/OperationCode/resources_api/blob/b6104234e3e5e2004b7cb285747815a44aaf29fc/tests/unit/test_models.py#L24-L37

But I don't like that this breaks the convention of everything else in the file. Perhaps we can find a way to inject it after the fact so it doesn't look out of place here?

Reply:
I made another property that calls serialize and adds the objectID to it.